internal/template/funcmap.go

```diff
@@ -90,6 +90,18 @@ func hclField(m map[string]interface{}, key string, req bool) (string, error) {
 
 // resourceName builds a Terraform resource name.
 // GCP resource names often use "-" but Terraform resource names should use "_".
-func resourceName(s string) string {
-	return strings.Replace(s, "-", "_", -1)
+func resourceName(m map[string]interface{}, k string) (string, error) {
```

Comment:
Maybe update the function comment now that it does more.

Reply:
Good point, done